src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -239,6 +272,7 @@ const ReviewStep = ({ data, onSubmit, onBack, setStep }: StepRenderProps<NewSafe
             },
           },
           data.safeVersion,
```

Comment:
We have to adjust the available `saltNonce` computation as the nonce has to be available on all selected chains and also not exist on any other chains yet.

Reply:
I adjusted the `saltNonce` computation in https://github.com/safe-global/safe-wallet-web/pull/4075/commits/8aa29c6c382cda64c105abefe1bb6ea8b6042d82